Sources/YumemiWeather/YumemiWeather.swift

```diff
@@ -183,14 +185,14 @@ final public class YumemiWeather {
     /// - Parameters:
     ///   - jsonString: 地域と日付を含む JSON 文字列
     ///   - completion: 完了コールバック
-    public static func callbackFetchWeather(_ jsonString: String, completion: @escaping (Result<String, YumemiWeatherError>) -> Void) {
+    public static func fetchWeather(_ jsonString: String, completion: @escaping (Result<String, YumemiWeatherError>) -> Void) {
         DispatchQueue.global().asyncAfter(deadline: .now() + apiDuration) {
             do {
                 let response = try fetchWeather(jsonString)
-                completion(Result.success(response))
+                completion(.success(response))
             }
-            catch let error where error is YumemiWeatherError {
-                completion(Result.failure(error as! YumemiWeatherError))
+            catch let error as YumemiWeatherError {
+                completion(.failure(error))
             }
             catch {
                 fatalError()
```

Comment:
せっかくなので、通常は有り得ないとはいえ無言で `fatalError` するのは控えておくと良いかもしれません。

```suggestion
                fatalError("An unexpected error has occurred: \(error.localizedDescription)")
```